src/main/java/core/basesyntax/model/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random randomizer = new Random();
+
+    public String getRandomColor() {
+        int numberOfColors = Color.values().length;
```

Comment:
Numbers of colors will be the same every time you call this method, maybe it makes sence to define this variable somewhere else?